Scripts/GetFieldsByIncidentType/GetFieldsByIncidentType.py

```diff
@@ -0,0 +1,79 @@
+import demistomock as demisto
+from CommonServerPython import *
+from pprint import pformat
+
+
+def get_field_by_long_name(fields, name):
+    for field in fields:
+        if field['name'] == name:
+            return field
+    return None
+
+
+def get_field_by_short_name(fields, name):
+    for field in fields:
+        if field['cliName'] == name:
+            return field
+    return None
```

Comment:
seems these functions are never used

Reply:
Yep, leftover from an old version and now removed